server/gitlab/types.go

```diff
@@ -0,0 +1,37 @@
+package gitlab
+
+import (
+	"time"
+
+	internGitlab "github.com/xanzy/go-gitlab"
+)
+
+type TriggerPipelineResponse struct {
+	ID          int        `json:"id"`
+	Status      string     `json:"status"`
+	Ref         string     `json:"ref"`
+	Tag         bool       `json:"tag"`
+	Username    string     `json:"username"`
+	Name        string     `json:"name"`
+	UpdatedAt   *time.Time `json:"updated_at"`
+	CommittedAt *time.Time `json:"committed_at"`
+	WebURL      string     `json:"web_url"`
+}
+
+func newTriggerPipelineResponse(pipeline *internGitlab.Pipeline) *TriggerPipelineResponse {
```

Comment:
I think `internGitlab` is supposed to be the `server/gitlab` package and _not_ the `go-gitlab` package. Also, see the previous comment about removing this file